cpp/cmake/templates/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
super dumb question: I don't see this file being included anywhere in the top-level cmake file. How then are these Find* modules getting generated?

Reply:
that happens here: https://github.com/dantegd/cuml/blob/a1b75f7ab0e9902d6135eaf74441ac2494ed5f55/cpp/CMakeLists.txt#L186

Reply:
Ah.. missed this statement. Thanks Dante!